### PR TITLE
Changes to config and manifest for deploying on govcloud

### PR DESCRIPTION
**Pull request by donjo**
This fixes the URL and changes the directory where static assets are served so the page loads.

### CODE REVIEW

**Review by fureigh — APPROVED** [review]
LGTM!

File: config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseurl = "https://ffd-microsite-staging.apps.cloud.gov/"
+baseurl = "https://ffd-microsite-staging.app.cloud.gov/"
```
Comment (fureigh) [inline config.toml:1]: @donjo This is meant to be staging, correct?

Comment (donjo) [reply config.toml:1]: correct. there's probably a better way to do this where both the staging and master are defined in the same config but as it's set up now they need the full baseurl defined to ensure the static assets load correctly in their respective builds/branches. I'll check in with you directly to talk about some better ways to do this.

File: .travis.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ script:
 - npm test
 - npm run build
 env:
-  - SITE_BASEURL_PRODUCTION="https://labs.usa.gov/" SITE_BASEURL_STAGING="https://ffd-microsite-staging.apps.cloud.gov/"
+  - SITE_BASEURL_PRODUCTION="https://ffd-microsite-production.apps.cloud.gov/" SITE_BASEURL_STAGING="https://ffd-microsite-staging.apps.cloud.gov/"
```
Comment (fureigh) [inline .travis.yml]: Thanks, @donjo! Two questions:
1) Shouldn't these be `.app.cloud.gov`?
2) Are they meant to be on the same line?

**Review by fureigh — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (donjo) [reply .travis.yml]: 1. doh! yes.
2. not sure. they were before I got here but unless this breaks something I'll change it to two lines for readability.

Comment (donjo) [reply .travis.yml]: Tried separating to different lines but it seems like travis parsed things differently and one of the build was failing.

**Review by fureigh — APPROVED** [review]
Hmm. Okay! Perhaps open a tiny `help wanted` ticket as follow-up, in case someone has time to see why Travis was objecting. But I'm fine with this as-is.

**Pull request merged**